router/pkg/config/config.schema.json

```
@@ -1647,6 +1647,57 @@
         }
       }
     },
+    "cache_warmup": {
+      "type": "object",
+        "description": "Cache Warmup pre-warms all caches (e.g. normalization, validation, planning) before accepting traffic.",
+        "additionalProperties": false,
+        "properties": {
+          "enabled": {
+              "type": "boolean",
+              "description": "Enable the cache warmup.",
+              "default": false
+          },
+          "source": {
+            "type": "string",
+            "description": "The source of the cache warmup items can be filesystem, cdn (Cosmo), or s3.",
+            "enum": ["filesystem", "cdn","s3"]
```

If we only support filesystem, we can omit cdn and s3 to fail fast.